code_samples/org-website/GlobalSerializerSample.js

```diff
@@ -22,12 +22,12 @@ class GlobalSerializer {
         this.id = 20;
     }
 
-    read(input) {
+    read(input) { // eslint-disable-line no-unused-vars
```

Comment:
Assuming this code sample be visible in the `.org` site, should we display these ignore comments? I know that it makes not much sense but maybe we can use JSON.parse/stringify here to demonstrate the concept?

Reply:
Good suggestion. Change the sample in 13cc25c0903a7d4d671c49bd1751eac20f1b161e